# Coding-the-central-dogma
The repo hosts a simple program in python which is efficient in generating an amino acid sequence from a fasta file input. It has 3 functions: 1) finding complementary sequence of the input fasta file 2) finding mRNA sequence of the complementary DNA sequence 3) Converting mRNA sequence to proteins contained in a higher-order class which results in output of a protein sequence. Finally, I used the package **Minotaor** to annotate the protein sequence which determines the important regions- start, stop, linker amino acids, histone tags, etc in the provided protein.The program finds its use in streamlining the process of converting genetic data to functional protein information, enhancing data analysis efficiency in genomic research. It can also enable precise annotation of protein sequences, aiding in the identification of functionally significant regions, which is crucial for bioinformatics research, drug discovery, and molecular biology studies. It provides a reliable tool for genomic and proteomic analyses, facilitating advancements in understanding gene expression and protein function.   
